radis/test/lbl/test_factory.py

```diff
@@ -725,6 +725,57 @@ def test_diluents_molefraction(verbose=True, plot=False, *args, **kwargs):
     sf.eq_spectrum(Tgas=300, mole_fraction=0.4)
 
 
+def test_vaex_and_pandas_spectrum():
+    from radis import calc_spectrum
+    from radis.spectrum.utils import CONVOLUTED_QUANTITIES
+
+    databanks = ["hitran", "hitemp", "exomol", "geisa"]
```

Comment:
why are you testing for multiple databases ?

Reply:
To ensure that some commit has not changed vaex and pandas spectrum for some databanks and spectrum is same.

Reply:
Let me challenge this. I did not think the changes your made were dependant on the database, are they ?

By doing it you're increasing downloads & computation times for the test suite, which is both environmentally problematic and slightly annoying for the users => we want tests to be as minimal and as fast as possible.
However, we want even more the test suite to be robust and trustworthy.

Therefore, if there is **one** good reason to test Vaex vs Pandas for different databases (i.e. that Vaex & Pandas might be equal for HITRAN, but different for HITEMP), let's keep it. But from my point of view your work is database-indepednant. 
So just use one database, and make sure the database is already used in other tests so we do not have to download one more database. 


Tagging @minouHub too ; I think it's important to maintain a clean & efficient test suite